.pre-commit-config.yaml

```diff
@@ -17,8 +17,6 @@ repos:
     rev: v2.2.3  # Use the ref you want to point at
     hooks:
     - id: mixed-line-ending
-      args:
```

Comment:
Not needed anymore ?

Reply:
This kept messing with my commits on windows, and I don't understand the point of it. Simply using `mixed-line-ending` should be enough right?

Reply:
By the way, the default is `auto`: https://github.com/pre-commit/pre-commit-hooks#mixed-line-ending

Reply:
It's probably an issue with your git config on windows, you should customize `core.autocrlf=false` (using `true` make git convert back and forth the encoding before doing the commit, hence having the pre-commit complaining given it does it check before the conversion happens)

I think we should keep this check because it's far too easy to add `CRLF` otherwise (typically when creating a new file on windows, which is not fixed by `mixed-line-ending=auto`)

Reply:
One possible solution would be to do `mixed-line-ending=auto` on local and `mixed-line-ending=lf` in the CI (it should be fairly easy to do that by adding a sed in the azure script)

Reply:
Done! 4149287